packages/editor/src/dataviews/store/private-actions.ts

```diff
@@ -133,19 +133,27 @@ export const registerPostTypeSchema =
 			.resolveSelect( coreStore )
 			.getCurrentTheme();
 
+		let canDuplicate =
+			! [ 'wp_block', 'wp_template_part' ].includes(
+				postTypeConfig.slug
+			) &&
+			canCreate &&
+			duplicatePost;
+
+		// @ts-ignore
+		if ( ! globalThis.IS_GUTENBERG_PLUGIN ) {
+			if ( postType === 'page' ) {
+				canDuplicate = undefined;
+			}
+		}
+
 		const actions = [
 			postTypeConfig.viewable ? viewPost : undefined,
 			!! postTypeConfig.supports?.revisions
 				? viewPostRevisions
 				: undefined,
 			// @ts-ignore
-			globalThis.IS_GUTENBERG_PLUGIN
-				? ! [ 'wp_block', 'wp_template_part' ].includes(
-						postTypeConfig.slug
-				  ) &&
-				  canCreate &&
-				  duplicatePost
-				: undefined,
+			canDuplicate,
```

Comment:
Wouldn't this make the duplicate action available for any post type (but `wp_block`, `wp_template_part`, `page`) in WordPress?

Note these actions are also used in the inspector of the post editor:

<img width="360" height="1242" alt="Screenshot 2025-10-21 at 11 28 52" src="https://github.com/user-attachments/assets/28e13345-a0d0-4737-9456-9d82d6121f94" />

We could enabling it just for templates, regardless of environment (gutenberg/wordpress).

Reply:
Oh, I didn't know this affects the post editor